core_backend/app/llm_call/dashboard.py

```diff
@@ -19,6 +22,11 @@ async def generate_ai_summary(
     """
     Generates AI summary for the dashboard.
     """
+    if DISABLE_DASHBOARD_LLM:
+        logger.info("LLM functionality is disabled. Returning default message.")
+        return """LLM functionality disabled on the backend. Please check your
```

Comment:
I think logging is fine but I am not sure we should return a string when it's disabled. That would confuse the users since they can't easily tell between actual AI summary and the error text. Let's just return `None`. What do you think?

Reply:
The useful thing is that this gets shown in the front-end. So if people are exploring their topics I think it makes more sense to tell them that it's disabled than for them to just see an empty box.

Reply:
But is an empty string expected when the LLM is on? 
If that's the case, writing that makes sense. If not we should just return None and on the frontend decide what to tell users. For example, for some use cases, the LLM being off is a mistake and for some use cases it is a feature. And the message should change depending on whether it is a mistake or a feature. 
So my suggestion would be :
- Return None
-  Add a message on frontend depending on whether it is an expected feature or not.